src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Sso/Client/UpdateClientDialog.razor.cs

```diff
@@ -22,20 +24,24 @@ public partial class UpdateClientDialog
     ClientAuthenticationDto _authenticationDto = new();
     ClientDeviceFlowDto _deviceFlowDto = new();
     ClientTokenDto _tokenDto = new();
+    ClientCredentialDto _clientCredentialDto = new();
+    Type _otherType = null!;
+
+    Dictionary<string, ComponentMetadata> _componentParameters = new();
```

Comment:
Why _componentParameters have a blank line above.

Reply:
Dictionary<string, ComponentMetadata> is already to Dictionary<string, object> and delete blank line